src/community/backup-restore/core/src/main/java/org/geoserver/backuprestore/utils/BackupUtils.java

```diff
@@ -89,10 +88,10 @@ private static Resource createRandomResource(String tempPath) throws IOException
         Resource directory = Resources.createRandom("tmp", "", root);
 
         do {
-            FileUtils.forceDelete(directory.dir());
+            directory.delete();
         } while (Resources.exists(directory));
 
-        FileUtils.forceMkdir(directory.dir());
```

Comment:
is there any difference beween these two methods?

Reply:
Yeah one is no longer available :) That said it did have slightly more robust deleting on windows.